src/app/lib/fonts.js

```diff
@@ -1,31 +1,6 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
There's this separation of fonts and styles - I think they might not be needed, since when we add fonts for other products, e.g. Persian, we'll need to have conditional logic depending on product in both font file and styles file.

Reply:
If I have understood you correctly, I don't think this conditional logic should live in either of these files. It should live higher up, and will determine what styles/fonts will be used.

Reply:
Agreed. The mapping can live in another file, within the `lib` directory.

Reply:
Reverted change